doc/internal/how_to_release.txt

```diff
@@ -64,11 +64,20 @@ The documentation will be automatically regenerated in the next few hours.
 Generate and upload the package
 ===============================
 
-For release candidates, only upload on PyPI.
+For non-final releases (e.g., release candidates, alpha, beta), upload on PyPI only.
```

Comment:
For release candidate, we should also email theano-user and theano-dev. For the final, we what other venue + theano-announce.